### PR TITLE
#165017740 Deploy app to heroku

### PR DESCRIPTION
**Pull request by CryceTruly**
**What does this PR do**
This PR enables users to access the live application from anywhere in the world.

**Description of tasks to be completed**
- Create a basic express server to serve the static react app
- Enable review apps on Heroku

**How can this be tested??**
- Visit the Current Review app https://ah-aquaman-frontend-stagi-pr-2.herokuapp.com/ in the browser
- Visit the live develop branch https://ah-aquaman-frontend-staging.herokuapp.com/ in your browser 
- Production app will be at https://ah-aquaman-frontend.herokuapp.com/

**Any background context you want to provide**
-  The develop branch is not live yet untill this task is merged to it.

**What are the relevant pivotal tracker stories**

- [#165017740](https://www.pivotaltracker.com/story/show/165017740)

### CODE REVIEW

**Review by Peace-Apple — APPROVED** [review]
Good work @CryceTruly, the review app works fine.

File: package.json [hunk 1]
```diff
@@ -2,13 +2,14 @@
   "name": "ah-aquaman-frontend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "app.js",
```
Comment (BagzieGracious) [inline package.json]: Thank you @CryceTruly for good work, can you please change `app.js` to `index.js` Since we have an `App.js` file in a src folder. This might bring up a conflict.

**Review by wasibani-roy — APPROVED** [review]

**Review by BagzieGracious — APPROVED** [review]
Thanks you @CryceTruly, for your good work

File: server.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const path = require("path");
+
+app.use(express.static('dist'));
+app.get("*", (req, res) => {
+ res.sendFile(path.resolve(__dirname, 'dist', 'index.html'));
+
+});
+
+const PORT = process.env.PORT || 5000;
+app.listen(PORT, () => console.log(`Server running on port ${PORT}`))
```
Comment (AmosWels) [inline server.js:12]: check this `console.log`, it's causing eslint to throw warnings.

File: package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "ah-aquaman-frontend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "src/App.js",
```
Comment (AmosWels) [inline package.json]: Why did you change the main from `index.js`

**Review by AmosWels — CHANGES_REQUESTED** [review]
Good work @CryceTruly, please resolve the eslint issues and a few reviews on your PR.

File: package.json (hunk 3 above)
Comment (CryceTruly) [reply package.json]: Actually, this is not correct, thank you @AmosWels for pointing it out should be 
` "src/index.js" `

**Review by AmosWels — APPROVED** [review]
